packages/pytest-simcore/src/pytest_simcore/docker_registry.py

```diff
@@ -285,3 +293,17 @@ def dy_static_file_server_dynamic_sidecar_compose_spec_service(
         docker_registry,
         node_meta_schema,
     )
+
+
+@pytest.fixture
+def remove_images_from_host() -> Callable[[list[str]], Awaitable[None]]:
+    async def _cleaner(images: list[str]) -> None:
+        print(f"--> Removing {images}")
+        async with aiodocker.Docker() as client:

```

Comment:
Q: AFAIK we are using both `docker` and `aiodocker`. Should we priorize using the latter?

Reply:
hmm... actually did not know when I started I would have also to move the log_context away from playwright and then fix playwright. hem. but ok